src/hmem_ze.c

```diff
@@ -309,8 +309,12 @@ ze_result_t ofi_zeDeviceGetProperties(ze_device_handle_t hDevice,
 	return (*libze_ops.zeDeviceGetProperties)(hDevice, pDeviceProperties);
 }
 
+#if HAVE_DRM || HAVE_LIBDRM
```

Comment:
Might be cleaner as 
```
#if HAVE_DRM
#include <drm/i915_drm.h>
#elif HAVE_LIBDRM
#include <libdrm/i915_drm.h>
#endif
```

Reply:
this `#if` covers much more than the `#include` lines.